selenium/src/test/java/au/org/emii/portal/tests/landing/AodnLinkTest.java

```diff
@@ -0,0 +1,14 @@
+package au.org.emii.portal.tests.landing;
+
+import au.org.emii.portal.tests.BaseTest;
+import org.testng.annotations.Test;
+
+public class AodnLinkTest extends BaseTest {
+
+    @Test
+    public void linkTest() {
+        webElementUtil.clickLinkContainingText("AODN");
+        webElementUtil.switchToNewTab();
+        webElementUtil.verifyPageTitle("Australian Ocean Data Network: IMOS.org.au");
```

Comment:
These tests are intended for portal-edge(for now). We can also run it against dev environment if we want.